internal/org/repository/eventsourcing/model/org_test.go

```diff
@@ -93,13 +93,6 @@ func TestAppendEvent(t *testing.T) {
 			},
 			result: &Org{ObjectRoot: es_models.ObjectRoot{AggregateID: "ID"}, State: int32(model.OrgStateActive)},
 		},
-		{
```

Comment:
why is this test deleted?

Reply:
because it doesnt work like this anymore. its a test from where we had the domain directly on the org.now its a sub element